src/nvim/tui/input_defs.h

```diff
@@ -0,0 +1,92 @@
+#ifndef NVIM_TUI_INPUT_DEFS_H
+#define NVIM_TUI_INPUT_DEFS_H
+
+typedef enum {
+  KITTY_KEY_CAPS_LOCK            = 57358,
```

Comment:
Not excited about adding lots of special case code for kitty. Are we going to be stuck with this forever or is it possible that libvterm would support it?

Reply:
Do you mean libtermkey?

Reply:
I meant `libtickit` actually :) https://www.leonerd.org.uk/code/libtermkey/